acceptance/acceptance_test.go

```diff
@@ -38,7 +38,16 @@ func TestAccept(t *testing.T) {
 	cwd, err := os.Getwd()
 	require.NoError(t, err)
 
-	execPath := BuildCLI(t, cwd)
+	coverDir := os.Getenv("CLI_GOCOVERDIR")
+
+	if coverDir != "" {
+		require.NoError(t, os.MkdirAll(coverDir, os.ModePerm))
```

Comment:
`os.ModePerm` expands to `0o777` -- intentional?

Reply:
yes, does not matter here.